README.md

```diff
@@ -94,9 +94,9 @@ And then run `./run.sh` again.
 
 ## Install the simulation tools locally
 
-The WebbPSF point-spread function model and Pandeia exposure time calculator engine are currently available for local installation by members of the science community. The required packages are distributed as part of Astroconda, a suite of astronomy-focused software packages for use with the [conda](https://conda.io/docs/) package manager for macOS and Linux.
+STIPS, the Space Telescope Image Product Simulator, is available at https://github.com/spacetelescope/STScI-STIPS.
```

Comment:
I actually meant to move the new STIPS sentence to the end of the very first paragraph in the README- which would become (with quite a few modifications I'm making here, while we are at it):

The WFIRST team at STScI has developed an exposure time calculator, a PSF model, and an image simulator for the science community to plan how they will use WFIRST. These tools are available separately as the Pandeia exposure time calculator engine, the WebbPSF point spread function modeling package, and the Space Telescope Image Product Simulator (STIPS).   Comprehensive setup documentation for local installation as well as tutorials for Pandeia and WebbPSF are provided here.  STIPS is available at https://github.com/spacetelescope/STScI-STIPS.  High level overviews of the functionality of the tools are available on [STScI's WFIRST website](http://www.stsci.edu/scientific-community/wide-field-infrared-survey-telescope/science-planning-toolbox).